gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemIntegrationTest.java

```diff
@@ -264,6 +264,28 @@ public void open_throwsExceptionWhenHadoopPathNull() {
     assertThat(exception).hasMessageThat().startsWith("hadoopPath must not be null");
   }
 
+  @Test
+  public void open_storageStatistics() throws IOException {
+    GoogleHadoopFileSystem myGhfs = createInMemoryGoogleHadoopFileSystem();
+
+    Path testRoot = new Path("/directory1/");
+    myGhfs.mkdirs(testRoot);
+
+    FSDataOutputStream fout = myGhfs.create(new Path("/directory1/file1"));
+
+    fout.writeBytes("data");
+    fout.close();
+    myGhfs.open(new Path("/directory1/file1"));
+    GhfsStorageStatistics ss1 = myGhfs.getStorageStatistics();
+    System.out.println("STORAGE STATS ***");
+    System.out.println(ss1.getLong("op_create"));
+    System.out.println(ss1.isTracked("op_list_located_status"));
+    System.out.println(ss1.getLong("op_list_located_status"));
+    System.out.println(ss1.isTracked("op_open"));
+    System.out.println(ss1.getLong("op_open"));
+    assertThat(myGhfs.delete(testRoot, /* recursive= */ true)).isTrue();
```

Comment:
Please add assertions for storage statistics

Reply:
Added